plugins/UsersManager/vue/src/PagedUsersList/PagedUsersList.vue

```diff
@@ -516,6 +530,9 @@ export default defineComponent({
   },
   data(): PagedUsersListState {
     return {
+      sortLastSeenAsc: true,
+      sortedByLastSeen: false,
+      filteredUsers: [],
```

Comment:
You have introduced a new `filteredUsers` object. But that one is actually not used anywhere. So you are sorting a list, that isn't displayed anywhere.